modules/cli/src/main/kotlin/datamaintain/cli/app/App.kt

```diff
@@ -21,20 +21,34 @@ import java.util.*
 
 class App : CliktCommand() {
 
-    private val configFilePath: File? by optionWithExample(
-        help = "path to config file",
-        example = "myProject/src/main/resources/config/datamaintain.properties")
+    private val configFilePath: File? by detailedOption(
+        help = "Path to config file. File must exist.",
+        example = "myProject/src/main/resources/config/datamaintain.properties"
+    )
         .convert { File(it) }
         .validate { it.exists() }
 
-    private val dbType: String? by option(help = "db type")
+    private val dbType: String? by detailedOption(
+        help = "db type",
+        defaultValue = CoreConfigKey.DB_TYPE.default
+    )
         .choice(DbType.values().associate { v -> v.value to v.value })
 
-    private val dbUri: String? by option(help = "mongo uri with at least database name. Ex: mongodb://localhost:27017/newName")
-
-    private val trustUri: Boolean? by option(help = "Deactivate all controls on the URI you provide Datamaintain").flag()
-
-    private val mongoTmpPath: String? by option(help = "mongo tmp file path")
+    private val dbUri: String? by detailedOption(
+        help = "mongo uri with at least database name. Ex: mongodb://localhost:27017/newName",
```

Comment:
You can remove the example part of the help description.

Reply:
Since the `--help` does not give you the details set here, I prefer to keep it to help users that use the CLI without reading the documentation